src/bus/webSocketConnection.js

```diff
@@ -107,6 +98,34 @@ class WebSocketConnection {
     this._webSocket.close();
   }
 
+  /**
+   * Handles WebSocket errors.
+   * The `ws` library also closes the socket when an error occurs.
+   * Since the socket connection closes, the jsonRpcId and message handlers are cleared
+   */
+  onError = (error) => {
+    this._messageHandlers = {};
+
+    console.log(error);
```

Comment:
It would be good to provide some context here, like `Error from this bus thing: ${error}`